src/app/check-out/check-out.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-check-out',
+  templateUrl: './check-out.component.html'
+})
+export class CheckOutComponent implements OnInit {
+  constructor() {}
```

Comment:
don't check in things that you are not using. It pollutes the code. Remove the constructor and the implements onInit and well as the ngOnInit()